src/translator/response_builder.cpp

```diff
@@ -5,11 +5,11 @@ namespace marian {
 namespace bergamot {
 
 void ResponseBuilder::buildQualityScores(Histories &histories,
-                                         Response &response) {
+                                         Response &response, int beamSize) {
   std::vector<Quality> qualityScores;
   for (auto &history : histories) {
-    // TODO(jerin): Change hardcode of nBest = 1
-    NBestList onebest = history->nBest(1);
+    //Due to inference performance, this should always be 1
+    NBestList onebest = history->nBest(beamSize);
```

Comment:
Restore to old state. I'm not sure what we gain by this parameterization of `beamSize` - could you elaborate why this was done?

Reply:
I think Marina was doing something with comparing the outputs of beam search.  Should really document what the intent is here.

Reply:
I'll have more info after Wednesday, but the primary reason to set this a parameter was just to use variable names instead of hard-coded code. I've set it as with a default value of 1 in `response_builder.h` in order to not change the behavior of the function. 

wdyt?